packages/components/src/pages/ChatSearch.vue

```diff
@@ -195,9 +197,20 @@ export default {
         explain: 'Explaining with AI',
       },
       ask: undefined,
+      loadAppMapStats: debounce(async () => {
+        const rpc = this.rpcClient();
+        try {
+          this.appmapStats = await rpc.appmapStats();
+        } catch (e) {
+          console.error('Error loading appmap stats', e);
+        }
+      }, 1000),
```

Comment:
The AppMap stats can be expensive to compute. I'd suggest a longer delay, maybe 5 seconds. Or, it should be adaptive to however long the stats took to compute the last time. Requesting stats every 1 second could be punishing for large projects.